src/main/native/jgskit.win64.mak

```diff
@@ -54,38 +56,49 @@ TARGET = $(HOSTOUT)/libjgskit_64.dll
 JGSKIT_RC_SRC = jgskit_resource.rc
 JGSKIT_RC_OBJ = $(HOSTOUT)/jgskit_resource.res
 
-all: headers $(TARGET)
-
-noheaders: $(TARGET)
-
-dircreate:
-	-@mkdir -p $(HOSTOUT) 2>nul
-
-headers: dircreate
-	$(JAVA_HOME)/bin/javac \
-	--add-exports java.base/sun.security.util=openjceplus \
-	--add-exports java.base/sun.security.util=ALL-UNNAMED \
-	$(TOPDIR)/src/main/java/com/ibm/crypto/plus/provider/ock/NativeInterface.java \
-	$(TOPDIR)/src/main/java/com/ibm/crypto/plus/provider/ock/FastJNIBuffer.java \
-	$(TOPDIR)/src/main/java/com/ibm/crypto/plus/provider/ock/OCKContext.java \
-	$(TOPDIR)/src/main/java/com/ibm/crypto/plus/provider/ock/OCKException.java \
-	-d $(JAVACLASSDIR) -h $(TOPDIR)/src/main/native/
+all: $(TARGET)
 
 $(TARGET): $(OBJS) $(JGSKIT_RC_OBJ)
 	link -dll -out:$@ $(OBJS) $(JGSKIT_RC_OBJ) -LIBPATH:"$(GSKIT_HOME)/lib" jgsk8iccs_64.lib
 
+$(JGSKIT_RC_OBJ) : $(JGSKIT_RC_SRC)
+	rc $(BUILD_CFLAGS) -Fo$@ $(JGSKIT_RC_SRC)
+
+$(HOSTOUT)/%.obj: %.c
```

Comment:
Some targets have a space before : and some do not. Do we want to make these consistant?